### PR TITLE
Update sef-team with available members

### LINKED ISSUE #703: Update sef-team
**Is your feature request related to a problem?**
We need to update sef-team with available members

**Describe the solution you'd like**
Add active members and remove inactive members

### PR DESCRIPTION
**Pull request by Gimhan-minion**
## Purpose
The purpose of this PR is to fix #703 

## Goals
To update sef team with active members.

## Approach
Add and remove the necessary members 
  
### Preview Link
https://pr-704-sef-site.surge.sh/team.html

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by akshika47 — DISMISSED** [review]

File: src/main/webapp/assets/content/json/team.json [hunk 1]
```diff
@@ -110,14 +110,6 @@
         "linkedin": "akshika"
       }
     },
-    {
-      "name": "Hasitha Senevirathne",
-      "description": "Deputy Chairperson",
-      "image": "hasitha.jpg",
```
Comment (anjula-sack) [inline src/main/webapp/assets/content/json/team.json:116]: Remove the image too.

**Review by anjula-sack — CHANGES_REQUESTED** [review]
Remove `hasitha.jpg` from the files.

File: src/main/webapp/assets/content/json/team.json (hunk 1 above)
Comment (Gimhan-minion) [reply src/main/webapp/assets/content/json/team.json:116]: Done

**Review by anjula-sack — APPROVED** [review]
Thank you! 😊 @Gimhan-minion

**Review by jayasanka-sack — APPROVED** [review]
Thanks @Gimhan-minion !

**Pull request merged**